src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+}
+
+.stars-container {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  flex-wrap: wrap;
+  width: 96px;
+  height: 96px;
+  margin: 2px;
+}
```

Comment:
These styles are unnecessary here. Let's delete them. You don't need to hardcode the stars' container width.
Cause of these styles - tests are failing

```suggestion
```